src/website/app/demos/Tooltip/bestPractices.js

```diff
@@ -0,0 +1,186 @@
+avoids a pain that produces no resultant pleasure?”
+`;
+
+const tooltipEveryFifthWord = words => {
+  function addSpaces(word, index) {
+    return (1 === index ? '' : ' ') + `${word} `;
+  }
+
+  return words.split(/[ \n]/).reduce((acc, word, index) => {
+    const possiblyTooltipedWord =
+      0 === index % 5 ? (
+        <Tooltip key={index} content={word} placement="top">
+          {word}
+        </Tooltip>
+      ) : (
+        addSpaces(word, index)
+      );
+    return acc.concat(possiblyTooltipedWord);
+  }, []);
+};
+
+export default [
+  {
+    type: 'do',
+    description:
+      'Use Tooltips to provide extra information, such as titles and keyboard shortcuts, for selected interface elements.',
+    example: (
+      <div>
+        <Tooltip content="Bold (⌘B)">
+          <Button minimal iconStart={<IconFormatBold title="Bold" />} />
+        </Tooltip>
+        <Tooltip content="Italic (⌘I)">
+          <Button minimal iconStart={<IconFormatItalic title="Italic" />} />
+        </Tooltip>
+        <Tooltip content="Underlined (⌘U)">
+          <Button
+            minimal
+            iconStart={<IconFormatUnderlined title="Underlined" />}
+          />
+        </Tooltip>
+        <Tooltip content="Left align (⌘+Shift+L)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignLeft title="Left align" />}
+          />
+        </Tooltip>
+        <Tooltip content="Center align (⌘+Shift+E)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignCenter title="Center align" />}
+          />
+        </Tooltip>
+        <Tooltip content="Right align (⌘+Shift+R)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignRight title="Right align" />}
+          />
+        </Tooltip>
+        <Tooltip content="Justify (⌘+Shift+J)">
+          <Button
+            minimal
+            iconStart={<IconFormatAlignJustify title="Justify" />}
+          />
+        </Tooltip>
+      </div>
+    )
+  },
+  {
+    type: 'dont',
+    description: `Don't use Tooltips to hide captions, subtitles, statuses, or secondaryText.
+Information hidden behind a hover interaction is much less likely to be discovered by the user.
+Important information should be displayed plainly on the page.
+`,
+    example: (
+      <DemoCardLayout>
+        <Card>
+          <CardTitle
+            actions={
+              <Tooltip content="Freidrich Mohs was a German geologist and mineralogist">
+                <IconHelp title="more info" />
+              </Tooltip>
+            }>
+            Friedrich Mohs
+          </CardTitle>
+          <CardBlock>
+            <b>Born</b>: January 29, 1773<br />
+            <b>Died</b>: September 29, 1839
+          </CardBlock>
+        </Card>
+      </DemoCardLayout>
+    )
+  },
+  {
+    type: 'dont',
+    description: `Don't put large strings of text inside a Tooltip's content. Tooltips don't scroll.`,
```

Comment:
Consider replacing that second sentence with "Tooltips are intended for short, simple content."?

Reply:
nice, I like that